crates/forge/src/init.rs

```diff
@@ -74,34 +74,31 @@ fn extend_gitignore(path: &Path) -> Result<()> {
 pub fn run(project_name: &str) -> Result<()> {
     let project_path = std::env::current_dir()?.join(project_name);
 
-    Command::new("scarb")
+    ScarbCommand::new_with_stdio()
         .current_dir(std::env::current_dir().context("Failed to get current directory")?)
         .arg("new")
         .arg(&project_path)
-        .stderr(Stdio::inherit())
-        .stdout(Stdio::inherit())
-        .output()
+        .run()
         .context("Failed to initialize a new project")?;
 
     let version = env!("CARGO_PKG_VERSION");
 
-    Command::new("scarb")
+    ScarbCommand::new_with_stdio()
         .current_dir(&project_path)
-        .arg("--offline")
+        .offline()
         .arg("add")
         .arg("snforge_std")
         .arg("--git")
         .arg("https://github.com/foundry-rs/starknet-foundry.git")
         .arg("--tag")
         .arg(format!("v{version}"))
-        .stderr(Stdio::inherit())
-        .stdout(Stdio::inherit())
-        .output()
+        .run()
         .context("Failed to add snforge_std")?;
 
-    let scarb_version = Command::new("scarb")
+    let scarb_version = ScarbCommand::new()
         .arg("--version")
-        .stderr(Stdio::inherit())
+        .inherit_stderr()
+        .command()
```

Comment:
Why `command().output()` instead of `run()`?

Reply:
I don't think we want the parsing of output to be a `ScarbCommand` responsibility. We could do `output` as a shortcut which runs `command().output()`.